src/Microsoft.OData.Core/JsonLight/ODataJsonLightCollectionSerializer.cs

```diff
@@ -50,6 +50,20 @@ internal void WriteCollectionStart(ODataCollectionStart collectionStart, IEdmTyp
                 // "@odata.context":...
                 this.WriteContextUriProperty(ODataPayloadKind.Collection, () => ODataContextUrlInfo.Create(collectionStart.SerializationInfo, itemTypeReference));
 
+                // "@odata.count":...

```

Comment:
@chinese007 Add some test cases. Also need to change the `WriteCollectionEnd`. De-serialize nextlink and count. Optional, depend on your user scenario, handle inline scenario